src/nccl_ofi_rdma.c

```diff
@@ -2315,7 +2319,11 @@ static inline int post_bounce_buffs_on_rail(nccl_net_ofi_rdma_ep_t *ep,
 			NCCL_OFI_WARN("Failed to allocate bounce req");
 			return -ENOMEM;
 		}
-		ret = send_progress(req);
+
+		assert(rail != NULL);
```

Comment:
`rail` must be not NULL as this point because we already dereferenced it at start of function and it isn't modified. You can remove this assert or move it to start of function.